docs/docs/associations.md

```diff
@@ -243,7 +243,7 @@ Project.belongsToMany(User, { through: UserProjects })
 To add a new project to a user and set it's status, you pass an extra object to the setter, which contains the attributes for the join table
 
 ```js
-user.addProject(project, { status: 'started' })
+User.addProject(project, { status: 'started' })
```

Comment:
Oh I thought it was model, forgot `addProject` can only be possible on `instance`. Will change